docker-compose.yml

```diff
@@ -108,7 +108,7 @@ services:
       TOKEN_AUTH_JWT_ALGO: ES256
       TOKENS_CONFIG: API_SERVICE_API_KEY:hex,AUTH_RESINOS_REGISTRY_CODE:hex,COOKIE_SESSION_SECRET:hex,JSON_WEB_TOKEN_SECRET:hex,MIXPANEL_TOKEN:hex,SUPERUSER_PASSWORD:hex,TOKEN_AUTH_BUILDER_TOKEN:hex,VPN_GUEST_API_KEY:hex,VPN_SERVICE_API_KEY:hex,API_VPN_SERVICE_API_KEY:API_SERVICE_API_KEY,REGISTRY2_TOKEN:TOKEN_AUTH_BUILDER_TOKEN
       TRUST_PROXY: 172.16.0.0/12
-      VPN_PORT: 443
+      VPN_PORT: ${VPN_PORT:-443}
```

Comment:
@matiasAS thank you, there are a few blockers **on our side** in relation to your PR, one I've already mentioned in this [comment](https://github.com/balena-io/open-balena/pull/271#issuecomment-2203948112) and the other is the way balena-cli currently handles env-var interpolation (different to compose). We are working on resolving both of these blockers and once we have a resolution, we should be able to hopefully merge this work.

Reply:
ok, is the blocking of the previous comments ok now?

Reply:
@ab77 
Sorry for asking again, do I have to do something else, or just wait for the other part of the environment variables to be resolved? Greetings

Reply:
Hello @ab77 , any progress with my suggestion of making the VPN port customizable?

Reply:
Great, now only the balena-ci blocker remains, this is on us @matiasAS.

Reply:
@ab77 
Excellent, I liked contributing to the project, even if it was a little small. 

I am developing on my own an api in fast api that communicates with the openbalena api in cases where at least 3 requests have to be made, everything is simplified with 1.

Reply:
Hello @ab77 , will my suggested change be applied one day or never?

Reply:
@matiasAS we can't merge this change until there is support for env-var interpolation at the balena-compose level.

Reply:
@ab77  do you like my proposal? Who does that depend on? I understand that it is not mandatory for my proposal to be accepted and perhaps I am the only openbalena user who needs it, I say this because if I manually modify docker compose and then do a git pull to update openbalena, an error will be generated in the merge, that is the only reason why I want you to apply that change

Reply:
It's rather more likely that we'd migrate ll of this to Kubernetes before balena-compose understands env. var. interpolation. In the mean time, just use `sed` or similar to patch the composition.

Reply:
> do you plan to implement more Balena Cloud functionalities in Openbalena? For example: the public url of the devices, SSH connection without doing the tunnel, update delta, etc.

No plans at present.

Reply:
@ab77 I am still on openbalena v3, and I have also tried v4, could there be breaking change updates that break the system?